stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -72,6 +200,127 @@ public View getViewForHighlighting(Object element) {
     return (View)element;
   }
 
+  @Override
+  protected void onGetStyles(View element, StyleAccumulator styles) {
+    for (Method method : getViewGetters()) {
```

Comment:
I think @jasta  and I are both of the idea that we really shouldn't be using reflection for this. Reflection just isn't fast, and in addition to that we're doing lots of processing and allocating and etc. etc.

Which methods do we really really need/want for this feature? Do we need all 70 of those? Or can we just manually hardcode a list of 20 or so? (in other words, instead of using a `Method[]` and going at it generically with reflection, we could just have explicit code for the N that we directly want to support)